xpi-manifest.yml

```diff
@@ -84,3 +84,12 @@ xpis:
       - web-ext-artifacts/cpmanager.xpi
     addon-type: mozillaonline-privileged
     install-type: npm
+  - name: pioneerv2example
+    description: Pioneer v2 Example
+    repo-prefix: pioneer-v2-example
```

Comment:
this is the line that needs to be `pioneerv2example`; the name can include dashes, but the repo-prefix needs to match the non-dash taskcluster.ci.config line.